packages/flutter/lib/src/material/data_table.dart

```diff
@@ -1087,6 +1099,7 @@ class DataTable extends StatelessWidget {
           onTapDown: cell.onTapDown,
           onSelectChanged: row.onSelectChanged == null ? null : () => row.onSelectChanged?.call(!row.selected),
           overlayColor: row.color ?? effectiveDataRowColor,
+          onRowLongPress: row.onLongPress == null ? null : () => row.onLongPress?.call(),
```

Comment:
Oh okay, to be honest I didn't really know why it was written that way so I just wanted to keep it similar, but now I get it! I changed it as suggested. Not sure what you mean about onEnter, for my use case I don't need it. If I wrote "drag through a range of rows" somewhere then I forgot about it. I only need to be able to long press on a row to mark it as selected, so that the checkboxes become visible, so that I can treat subsequent taps as selecting more rows (instead of opening the entry in a seperate window in my app).